Fashionz/index.html

```diff
@@ -0,0 +1,261 @@
+<!DOCTYPE html>
+
+<html>
+    <head>
+        <title>
+            Fashionz
+        </title>
+        <script src="https://kit.fontawesome.com/d95ab1ac46.js"></script>
+        <link rel="stylesheet" href="/Fashionz/assets/css/styles.css">
```

Comment:
@rubeshudayakumar link tags should be closed. Have a habit of closing all self closing tags